### DISCUSSION TITLE
How to transition YAML config to `3.0.0-beta1`?

### DISCUSSION THREAD

**Discussion by Desoroxxx** [apache/logging-log4j2#2143] [Q&A]
Since the version `3.0.0-beta1` removed the `log4j-layout-jackson-yaml` module, my configuration file does not work, that is somewhat expected.

But I am not quite sure of what should I do to transition to `3.0.0-beta1`, what should I do?

**Comment by vy** [comment]
@Desoroxxx, YAML configuration (e.g., `log4j2.yaml`) and YAML layout (i.e., printing log messages in YAML format) are two different things. We have removed the latter, the former should still work, if you have `com.fasterxml.jackson.dataformat:jackson-dataformat-yaml` in your classpath. Could you check this, please?

**Comment by ppkarwasz** [comment]
For the next beta I submitted #2142, which should simplify adding all the necessary dependencies to use YAML config files.
After the change, users will just have to add `log4j-config-yaml` to their runtime dependencies.
